File: app/src/main/java/com/ac/demo/NameTuple.java
package com.ac.demo;

import android.arch.persistence.room.ColumnInfo;

/**
 * Created by anna on 2017/11/29.
 */

public class NameTuple {

    @ColumnInfo(name="first_name")
    public String firstName;

    @ColumnInfo(name="last_name")
    public String lastName;
}

File: simple/src/main/java/com/kotlin/sample/persistence/user/NameTuple.kt
package com.kotlin.sample.persistence.user

import android.arch.persistence.room.ColumnInfo

/**
 * Created by anna on 17/01/2018.
 */
class NameTuple {
    @ColumnInfo(name = "first_name")
    var firstName: String? = ""

    @ColumnInfo(name = "last_name")
    var lastName: String? = ""
}